build.sbt

```diff
@@ -18,7 +18,7 @@ val sttpV                = "3.8.11"
 val kafkaV               = "3.3.1"
 val testContainersScalaV = "0.41.0"
 
-ThisBuild / version := "0.1-SNAPSHOT"
+ThisBuild / version := "1.18.0-SNAPSHOT"
```

Comment:
Shouldn't we use our [versioning approach](https://touk.slack.com/archives/C02MYBJAY10/p1725366943796189) also this case?